src/Cli/Commands/ConfigureOptions.cs

```diff
@@ -67,16 +67,16 @@ public ConfigureOptions(
         [Option("runtime.graphql.depth-limit", Required = false, HelpText = "Max allowed depth of the nested query. Allowed values: (0,2147483647] inclusive. Default is infinity. Use -1 to remove limit.")]
         public int? DepthLimit { get; }
 
-        [Option("runtime.graphql.enabled", Required = false, HelpText = "Configure enable/disable flag for graphQL runtime options.")]
+        [Option("runtime.graphql.enabled", Required = false, HelpText = "Enable DAB's GraphQL endpoint in Runtime Config. Allowed values: true(default) or false (boolean).")]
```

Comment:
nit: the phrase "in Runtime Config" seems unnecessary since contextually we know the command is going to affect the config file. `Enable GraphQL endpoint. Allowed values: true(default) or false (boolean).` seems sufficient.

Reply:
removed "in runtime config"